src/wp-includes/vars.php

```diff
@@ -146,7 +146,9 @@
  * @return bool
  */
 function wp_is_mobile() {
-	if ( empty( $_SERVER['HTTP_USER_AGENT'] ) ) {
+	if ( isset( $_SERVER['HTTP_SEC_CH_UA_MOBILE'] ) ) {
```

Comment:
Maybe add an inline comment above this pointing to some doc link about this?

The name of this header is quite cryptic, so would be great to have a reference handy.

Reply:
Done in 1383868e23